### PR TITLE
Add upgrade job ran through scale-ci-ansible

### PR DESCRIPTION
**Pull request by akrzos**
* Also fix scale job since location of playbook has shifted

### CODE REVIEW

File: dockerfiles/pbench/controller/run.sh [hunk 1]
```diff
@@ -71,7 +71,16 @@ elif [[ $JOB == "scaleup" ]]; then
...
+        source /opt/pbench-agent/profile; pbench-user-benchmark -- ansible-playbook -vvv -i /root/scale-ci-ansible/scaleup.inv OCP-4.X/scale.yml; pbench-move-results --prefix=scaleup
+elif [[ $JOB == "upgrade" ]]; then
+        echo "Running ocp upgrade"
+        git clone https://github.com/redhat-performance/scale-ci-ansible.git /root/scale-ci-ansible
+        cd /root/scale-ci-ansible
+	cat <<-EOF >> upgrade.inv
+	[orchestration]
+	localhost ansible_connection=local
+	EOF
+        source /opt/pbench-agent/profile; pbench-user-benchmark -- ansible-playbook -vvv -i /root/scale-ci-ansible/upgrade.inv OCP-4.X/upgrade.yml; pbench-move-results --prefix=scaleup
```
Comment (chaitanyaenr) [inline dockerfiles/pbench/controller/run.sh]: The prefix has to set to upgrade.

Comment (akrzos) [reply dockerfiles/pbench/controller/run.sh]: Thanks, fixed now.

**Comment by akrzos** [comment]
@chaitanyaenr Can we merge this to run with the smoke test?

**Comment by chaitanyaenr** [comment]
@akrzos I don't have the merge rights. The PR looks good to me, we can go ahead and merge it. Will bake this into the image.

**Pull request merged**